mcs/class/corlib/monotouch_corlib.dll.exclude.sources

```diff
@@ -1,3 +1,4 @@
+corefx/RandomNumberGeneratorImplementation.Mono.cs
```

Comment:
Why is corefx/Interop.GetRandomBytes.Mono.cs needed in this profile?

Reply:
This is the .exclude.